android-dictation-log.md

```
@@ -0,0 +1,134 @@
+- WhisperInput: https://github.com/alex-vt/WhisperInput
+- Study on ASR for Non-Native English: https://arxiv.org/abs/2503.06924
+- OpenAI Whisper Model: https://github.com/openai/whisper
+- Whisper.cpp: https://github.com/ggerganov/whisper.cpp
+- AssemblyAI: https://www.assemblyai.com/
+- Deepgram: https://www.deepgram.com/
+- Speechmatics: https://www.speechmatics.com/
+- Reddit discussion: https://www.reddit.com/r/androidapps/comments/1dkdy1m/alternative_keyboard_to_google_and_ms_that_still/
+
+---
+
+_End of Log_
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)